Gemfile

```diff
@@ -59,6 +59,7 @@ group :test do
   gem 'simplecov', require: false
   gem 'simplecov-console', require: false
   gem 'shoulda-matchers', '~> 3.1'
+  gem 'rails-controller-testing'
```

Comment:
“Controller specs can be used to describe the behaviour of Rails controllers. As of version 3.5, however, controller specs are discouraged in favour of request specs (which also focus largely on controllers, but capture other critical aspects of application behaviour as well). Controller specs will continue to be supported until at least version 4.0 (see the release notes for details).”

https://medium.com/just-tech/rspec-controller-or-request-specs-d93ef563ef11
https://github.com/rspec/rspec-rails#request-specs